sefaria/export.py

```diff
@@ -726,7 +726,11 @@ def _import_versions_from_csv(rows, columns, user_id):
     from sefaria.tracker import modify_bulk_text
 
     index_title = rows[0][columns[0]]  # assume the same index title for all
-    index_node = Ref(index_title).index_node
+    index = Index().load({'title': index_title})
+    if index:
+        index_node = index.nodes
```

Comment:
If we're getting `index.nodes`, which is plural, I would have assumed that it would be assigned to `index_nodes`, not `index_node`. Or does `index.nodes` returns only a single node?

Reply:
it returns the root node which must be single. i don't know why its name is plural, i guess it has some history.